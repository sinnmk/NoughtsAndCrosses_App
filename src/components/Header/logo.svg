 <svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="700.000000pt" height="700.000000pt" viewBox="0 0 700.000000 700.000000" preserveAspectRatio="xMidYMid meet">
<metadata>
Created by potrace 1.12, written by Peter Selinger 2001-2015
</metadata>
<g transform="translate(0.000000,700.000000) scale(0.100000,-0.100000)" fill="#000000" stroke="none">
<path d="M3320 6997 c-3 -3 -43 -8 -90 -11 -254 -18 -435 -51 -735 -134 -220
-61 -537 -199 -770 -335 -419 -245 -841 -642 -1126 -1062 -194 -285 -369 -653
-451 -950 -107 -388 -143 -640 -143 -1005 0 -182 4 -247 26 -445 20 -185 85
-473 148 -660 35 -104 51 -149 61 -170 5 -11 30 -69 57 -130 177 -407 403
-735 733 -1065 216 -216 380 -347 646 -516 95 -61 325 -179 454 -234 41 -17
84 -35 95 -40 179 -82 613 -188 858 -210 67 -6 156 -15 197 -20 97 -13 343
-13 440 0 41 5 131 14 200 20 214 19 563 100 760 175 19 8 64 25 100 39 636
245 1236 726 1620 1300 84 125 135 207 157 251 11 22 44 87 72 144 111 221
152 324 235 601 143 471 171 1087 74 1605 -11 60 -31 152 -45 205 -14 52 -30
118 -38 145 -7 28 -22 77 -34 110 -11 33 -28 80 -36 105 -9 25 -20 54 -25 65
-5 11 -23 54 -41 95 -57 136 -180 375 -240 467 -186 285 -303 429 -518 643
-318 316 -642 539 -1046 720 -121 54 -333 132 -420 155 -196 52 -253 65 -350
84 -99 18 -186 30 -355 47 -81 8 -464 17 -470 11z m-519 -1126 l24 -19 5 -743
5 -743 665 -1 665 0 5 743 5 744 24 19 c33 27 102 26 136 -3 l26 -23 2 -740 2
-740 738 0 739 0 25 -29 c21 -25 25 -37 20 -79 -8 -89 58 -81 -780 -87 l-742
-5 0 -665 0 -665 742 -5 c838 -6 772 2 780 -87 4 -39 1 -53 -17 -75 l-21 -28
-742 -2 -742 -3 -3 -742 -2 -742 -28 -21 c-22 -18 -36 -21 -75 -17 -89 8 -81
-58 -87 780 l-5 742 -665 0 -665 0 -5 -742 c-6 -838 2 -772 -87 -780 -42 -5
-54 -1 -79 20 l-29 25 -2 739 -2 738 -738 2 -738 2 -22 26 c-30 34 -31 103 -4
136 l19 24 742 5 741 5 0 665 0 665 -742 5 -741 5 -19 24 c-29 35 -26 105 6
136 30 31 -35 28 818 29 l678 1 2 738 2 739 28 24 c36 31 104 33 138 5z m-662
-351 c169 -66 286 -196 331 -367 24 -92 25 -149 5 -228 -53 -206 -194 -347
-400 -400 -32 -8 -80 -15 -105 -15 -55 0 -191 34 -228 57 -128 78 -143 91
-198 175 -71 108 -89 166 -89 288 0 108 12 157 64 258 27 52 102 134 157 171
45 29 154 75 204 85 55 11 205 -3 259 -24z m1549 -9 c130 -53 216 -130 272
-244 107 -220 63 -452 -115 -616 -71 -64 -132 -97 -224 -121 -90 -24 -147 -25
-226 -5 -191 50 -325 173 -392 362 -26 74 -24 220 5 305 56 161 160 267 324
327 75 28 92 30 193 26 72 -3 105 -10 163 -34z m1215 -210 l127 -129 129 132
c128 129 130 131 175 131 78 0 127 -75 93 -141 -8 -16 -69 -82 -135 -146
l-120 -118 123 -121 c68 -66 129 -134 135 -150 26 -61 -28 -139 -96 -139 -35
0 -47 10 -169 130 -72 72 -133 130 -136 130 -3 0 -62 -57 -130 -126 -112 -113
-129 -126 -167 -131 -38 -5 -45 -2 -77 30 -28 28 -35 43 -35 73 0 35 10 47
132 171 l132 133 -132 133 c-119 121 -132 137 -132 169 0 39 29 84 64 99 11 5
37 7 56 4 29 -4 56 -26 163 -134z m290 -1310 c76 -26 155 -80 217 -148 90 -98
132 -208 132 -343 0 -145 -46 -257 -151 -361 -107 -107 -215 -153 -361 -153
-100 -1 -158 14 -251 61 -147 73 -255 244 -266 421 -18 264 166 493 437 546
48 9 186 -4 243 -23z m-3356 -214 l132 -133 134 133 c148 147 163 154 228 110
41 -28 57 -74 39 -116 -7 -17 -68 -84 -135 -150 l-124 -120 130 -128 c109
-108 131 -134 136 -165 5 -34 2 -42 -36 -79 -37 -38 -45 -41 -78 -36 -31 5
-59 28 -166 135 l-128 128 -129 -130 c-128 -131 -129 -131 -175 -131 -40 0
-51 5 -76 33 -21 23 -29 43 -29 69 0 33 11 48 132 170 l132 133 -133 135 -134
134 6 41 c8 64 47 98 112 99 25 1 50 -19 162 -132z m283 -1315 c95 -33 143
-63 213 -133 72 -72 114 -147 142 -254 20 -79 19 -136 -5 -228 -46 -174 -167
-306 -338 -369 -86 -32 -244 -31 -332 2 -158 58 -263 163 -321 320 -31 86 -34
248 -5 325 56 146 123 224 259 303 106 61 268 75 387 34z m3019 12 c101 -25
178 -69 252 -143 104 -104 152 -219 153 -361 1 -170 -85 -332 -223 -424 -105
-70 -166 -89 -291 -90 -93 -1 -113 2 -177 27 -162 64 -259 164 -320 334 -24
66 -24 240 0 306 66 182 189 298 368 346 92 25 154 26 238 5z m-1766 -233
l127 -129 127 129 c146 149 167 159 233 115 40 -26 59 -72 46 -113 -3 -10 -64
-76 -134 -146 l-129 -128 131 -129 c130 -128 131 -129 131 -174 0 -59 -33 -95
-94 -103 l-42 -6 -134 134 -135 133 -133 -132 c-122 -121 -137 -132 -170 -132
-26 0 -46 8 -69 29 -28 25 -33 36 -33 76 0 46 0 47 131 175 l130 128 -134 136
-135 135 6 41 c9 68 53 103 117 95 29 -4 56 -26 163 -134z"></path>
<path d="M2835 4158 c-3 -7 -4 -308 -3 -668 l3 -655 665 0 665 0 0 665 0 665
-663 3 c-525 2 -664 0 -667 -10z m537 -385 l128 -127 133 132 c122 121 137
132 170 132 26 0 46 -8 69 -29 28 -25 33 -36 33 -75 0 -46 -1 -47 -127 -173
-71 -70 -128 -129 -128 -132 0 -3 57 -62 126 -132 110 -110 126 -131 131 -165
10 -77 -75 -136 -143 -101 -16 8 -82 69 -146 135 l-118 120 -129 -131 c-129
-131 -130 -132 -175 -132 -59 0 -95 33 -103 94 l-6 42 134 134 133 135 -129
130 c-127 127 -130 131 -130 174 0 69 47 110 115 101 28 -4 57 -27 162 -132z"></path>
<path d="M1865 5332 c-74 -24 -175 -127 -198 -203 -46 -151 12 -292 158 -380
35 -21 52 -24 140 -24 114 0 166 19 229 84 64 67 81 113 81 226 0 88 -3 105
-24 140 -87 145 -236 206 -386 157z"></path>
<path d="M3404 5331 c-59 -20 -100 -48 -143 -96 -42 -48 -57 -80 -71 -152 -28
-140 46 -276 183 -338 51 -24 68 -27 140 -23 91 3 147 26 205 81 99 95 125
253 61 378 -26 53 -111 125 -175 148 -66 25 -131 26 -200 2z"></path>
<path d="M4950 3806 c-63 -18 -99 -38 -144 -82 -47 -45 -85 -124 -93 -192 -17
-143 85 -296 221 -333 23 -6 77 -9 121 -7 89 5 143 29 202 90 61 64 78 112 78
218 0 80 -4 102 -23 140 -46 89 -124 149 -220 169 -64 13 -84 13 -142 -3z"></path>
<path d="M1870 2271 c-43 -14 -104 -56 -139 -97 -73 -83 -97 -191 -65 -297 15
-52 29 -72 84 -127 52 -52 76 -69 121 -83 151 -46 293 13 379 157 19 32 25 58
28 127 4 78 2 93 -22 146 -31 67 -72 113 -136 150 -38 23 -58 27 -135 30 -49
1 -101 -1 -115 -6z"></path>
<path d="M4930 2271 c-73 -22 -157 -100 -190 -174 -77 -174 13 -373 194 -430
148 -47 303 20 378 163 19 38 23 60 23 140 0 80 -4 102 -23 140 -34 66 -79
109 -147 141 -51 24 -72 29 -135 28 -41 0 -86 -4 -100 -8z"></path>
</g>
</svg>