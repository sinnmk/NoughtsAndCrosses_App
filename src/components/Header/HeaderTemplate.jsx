import React from 'react';
import logo from './logo.svg';

const HeaderTemplate= () =>{
    return(
        <div className="App-header">
            <img src={logo} className="App-logo" alt="logo" />
            <h1 className="App-title">TIC TAC TOE</h1>
        </div>
    )
}
export default HeaderTemplate;