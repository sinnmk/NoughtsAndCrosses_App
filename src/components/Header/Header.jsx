import {Component} from 'react';
import 'reactstrap'
import HeaderTemplate from './HeaderTemplate'

export default class Header extends Component{
    constructor(props){
        super(props);
        this.state = {}
    }

    render(){
        return HeaderTemplate.call(this);
    }
}