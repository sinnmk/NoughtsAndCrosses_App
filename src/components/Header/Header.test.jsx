import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header'

describe('When Header is rendered',() =>{

    test('Header displays Header content', () =>{
        const component = renderer.create(
            <Header/>).toJSON();

        expect(component).toMatchSnapshot();
    })

})