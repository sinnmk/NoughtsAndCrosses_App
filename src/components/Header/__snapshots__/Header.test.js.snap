// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When Header is rendered Header displays Header content 1`] = `
<div
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
  <h1
    className="App-title"
  >
    TIC TAC TOE
  </h1>
</div>
`;
